src/controllers/files/shared_file/_handler.py

```diff
@@ -0,0 +1,29 @@
+from src.config.soap_client import soap_client
+
+
+def shared_files_handler(token):
+    try:
+        request_data = {"token": token}
+        response = soap_client.service.share_list(request_data)
+
+        if response["error"] is True:
+            return {"msg": response["msg"]}, response["code"]
+
+        sharedFile = [
```

Comment:
`extension` field is missing according to the [gateway spec](https://editor.swagger.io/).